### DISCUSSION TITLE
Anyone know how to setup Blackhole on the MacBook Pro with M2 Chip

### DISCUSSION THREAD

**Discussion by Sageman7156** [ExistentialAudio/BlackHole#744] [Q&A]
Anyone know how to setup Blackhole (2-ch installed) on the MacBook Pro with an M2 Chip?  I have the Scarlett 18/20 interface and running Ventura 13.5.2 but don't see an audio midi setup option in preferences and don't see any way to create a multi-output device. 

Any help appreciated.

**Comment by devinroth** [comment]
Search in spotlight.

Devin

**Reply by Sageman7156** [reply]
Thanks... I found the midi audio setup and created the multi output channel... thought I was on the road but when I go to Ableton preferences, and use the Blackhole input and try to record, nothing is coming through.   

On system Preferences I have:
Output to Output Device      
Input to Scarlett 18/20

In Ableton:
Output to Blackhole  
Input to Scarlett 18/20

Is that right?

**Comment by devinroth** [comment]
System Pref. Output to Multi-output (BH + Speakers)
Abelton input as BlackHole. Then record.

Devin

**Reply by Sageman7156** [reply]
Still no joy...

Here is the current setup:

System Pref:
Input - BlackHole 2-ch  (There is no Multi Output device listed on the input side)
Output - Multi Output With BlackHole 2ch, MacBook Pro Speakers and Scarlett 18/20 checked

Ableton:
Input  - BlackHole 2-Ch
Output - Scarlett 18/20

I hit record and and start it running but still nothing coming into Ableton.

Anything look wrong or other suggestions??

**Reply by Sageman7156** [reply]
I also tried it with Logic and couldn't get any input into the DAW.  

Thanks for all the help.

**Reply by onButtonUp** [reply]
I got it to work on a M2 mac, logic pro, and motu m6 audio interface (using Sonoma).

<img width="1678" alt="image" src="https://github.com/ExistentialAudio/BlackHole/assets/7108393/464634da-c2a9-4b39-a7ae-c9e9118336fc">

And System Settings, Sound, Output -> BlackHole.

**Comment by antonyRock** [comment]
macbook m1 max 
mac os - Sonoma 14.1 (23B74)
OBS 30.0.0-rc2

same problem try to use sound from garage band to obs 
but sound from micro not come to obs

plz help

Garage band - https://disk.yandex.ru/i/rDner-SBfp9yTQ
OBS 1 https://disk.yandex.ru/i/rDner-SBfp9yTQ
OBS 2 https://disk.yandex.ru/i/PJiiI7vT0ABanw
MIDI - https://disk.yandex.ru/i/o_bFK9qfqoOiWg
System - https://disk.yandex.ru/i/rrDGQ3dP-HYz1Q

**Reply by Sageman7156** [reply]
I'm trying Caster from Ginger Audio to see if that will work.  It will be a couple of days before I can really try it out but I will comment on it here.

**Reply by Sageman7156** [reply]
Still no joy... hopefully someone will come out with a specific fix.

**Comment by devinroth** [comment]
Are you able to record audio from BlackHole with Audacity?

Devin

**Reply by Sageman7156** [reply]
I didn't try Audacity just Ableton and Logic Pro.  I'll give that a try.

**Comment by devinroth** [comment]
Also check BlackHole volume and mute.

Devin

**Reply by Sageman7156** [reply]
Will do.... thanks so much!!!

**Comment by lucid-llama** [comment]
No luck using bh 2ch as output from VLC 3.0.20 or quicktime to mac os notes with bh as mic input for dictation. Mac OS X Sonoma 14.3.1

When I play audio, I see the sound registering on the meter in sound system preferences and registering with bh mic, but when I press the dictation shortcut in notes, the mic no longer registers input. 

Any advice appreciated, but advice that resolves the problem especially appreciated ;) Tx

**Comment by devinroth** [comment]
That’s a completely different topic.

In short, Apple doesn’t want you to be using virtual audio devices as input into dictation.

Devin

**Comment by lucid-llama** [comment]
Rats!

But thanks for clarifying. That was helpful.

While this may not be the place, I could also ask if there's a way to circumvent the problem with any other method using the equipment mentioned. I'll try doing it with Google's service.

Cheers

**Comment by lucid-llama** [comment]
Yup, Blackhole as virtual output and virtual input works with Google's voice typing feature, should anyone wish to know. 

However, for some reason, Sonoma doesn't seem to support multiple simultaneous outputs. Perhaps I'm missing something there.
